src/openrct2-ui/windows/TileInspector.cpp

```diff
@@ -446,8 +446,7 @@ static bool windowTileInspectorTileSelected = false;
 static int32_t windowTileInspectorToolMouseX = 0;
 static int32_t windowTileInspectorToolMouseY = 0;
 static bool windowTileInspectorToolCtrlDown = false;
-static int32_t windowTileInspectorToolMapX = 0;
-static int32_t windowTileInspectorToolMapY = 0;
+static CoordsXY windowTileInspectorToolMap = {};
```

Comment:
Any reason to use CoordsXY instead of TileCoordsXY? The latter seems to be more appropriate for the Tile Inspector.

Reply:
Because i was standardising the game actions to use coordsxy